docs/2023/QuML/Enhanced Quml player for Enhanced Question types/updates/2023-08-11.md

```diff
@@ -4,13 +4,25 @@ author: Yashi Gupta
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Updation of Attachments section UI - Milestone 1
+- [] Updation of Attachments section Logical implementation - Milestone 2
```

Comment:
provide a space between square brackets to mark a milestone as an incomplete milestone or give x to mark  as completed

Reply:
gave the space in between the brackets.